formats/json/jsMain/src/kotlinx/serialization/json/internal/DynamicObjectSerializer.kt

```diff
@@ -71,6 +71,7 @@ private class DynamicObjectEncoder(
         OBJ, MAP, LIST
     }
 
+    @OptIn(ExperimentalSerializationApi::class)
```

Comment:
We have `file:OptIn`. Why it's not enough?

Reply:
Because IDE support is buggy